.github/workflows/android-dev-orange-deploy.yaml

```diff
@@ -0,0 +1,53 @@
+name: Android dev-orange deploy
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - synchronize
+      - labeled
+  schedule:
+    - cron: "13 12 * * MON"
```

Comment:
question (non-blocking): Why different cron than backend?

Reply:
I picked these times arbitrarily, and then I picked the backend times with more attention to normal working hours just in case something goes wrong, and then I did not update these to match that.